ORCA 6 implicit solvent

JDSomper opened the pull request:
ORCA 6.0 compatibility for implicit solvation


***

## Checklist

* [ ] The changes include an associated explanation of how/why
* [ ] Test pass
* [ ] Documentation has been updated
* [ ] Changelog has been updated

shoubhikraj committed:
fix solvent for ORCA 6

shoubhikraj committed:
bug fix

JDSomper committed:
Merge pull request #1 from shoubhikraj/solvent_fix

Move edits

shoubhikraj commented:
@JDSomper The tests seem to be failing (https://github.com/duartegroup/autodE/actions/runs/15853629556/job/44693272118?pr=386) because the ORCA version is converted to integer, but in tests the ORCA version cannot be detected so it just comes as "???".

The conversion to integer should be done with a try -- except probably.

JDSomper commented:
Thanks @shoubhikraj ! I'll get round to it but that's excellent to know!

JDSomper committed:
Update ORCA.py

Changed the is_v5 and is_v6 commands to not crash if no version of ORCA is provided.

JDSomper committed:
Update ORCA.py

Removed trailing whitespace
